src/js/components/Main/Home/Home.jsx

```diff
@@ -1,10 +1,143 @@
+                        <br/>
+                        <span>{languageSrc.place[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[0].place : ""}</b>
+                        <br/>
+                        <span>{languageSrc.date[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[0].date : ""}</b>
+                        <br/>
+                      </div>
+                    </div>
+                  </div>
+                </Col>
+                <Col sm={12} xs={12} md={12} lg={4} xl={4}>
+                  <div className={style.centreDiv}>
+                    <div className={style.frontImage}>
+                      <img className={style.centreImage}
+                           src={props.recent.length !== 0 ? props.recent[1].imgSrc : "https://media2.giphy.com/media/3oEjI6SIIHBdRxXI40/200.gif"}/>
+                    </div>
+                    <div className={style.backImage}>
+                      <div className={style.infoCol}>
+                        <span>{languageSrc.band[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[1].band : ""}</b>
+                        <br/>
+                        <span>{languageSrc.place[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[1].place : ""}</b>
+                        <br/>
+                        <span>{languageSrc.date[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[1].date : ""}</b>
+                        <br/>
+                      </div>
+                    </div>
+                  </div>
+                </Col>
+                <Col sm={12} xs={12} md={12} lg={4} xl={4}>
+                  <div className={style.rightDiv}>
+                    <div className={style.frontImage}>
+                      <img className={style.centreImage}
+                           src={props.recent.length !== 0 ? props.recent[2].imgSrc : "https://media2.giphy.com/media/3oEjI6SIIHBdRxXI40/200.gif"}/>
+                    </div>
+                    <div className={style.backImage}>
+                      <div className={style.infoCol}>
+                        <span>{languageSrc.band[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[2].band : ""}</b>
+                        <br/>
+                        <span>{languageSrc.place[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[2].place : ""}</b>
+                        <br/>
+                        <span>{languageSrc.date[langProps.language]}:</span>
+                        <b>{props.recent.length !== 0 ? props.recent[2].date : ""}</b>
+                        <br/>
+                      </div>
+                    </div>
+                  </div>
+                </Col>
```

Comment:
**General comment**: These columns are basically the same code repeated with different content. For example (it's a pseudocode. only like an idea):
```
const content = [
  {
    classNameDiv: 'leftDiv',
    ...,
  },
  {
   classNameDiv: 'centreDiv',
    ...,
 
  },
  ...
]
...
{content.map(tile =>
  <Col sm={12} xs={12} md={12} lg={4} xl={4}>
    <div className={style.tile.classNameDiv}>
      <div className={style.frontImage}>
        <img className={style.centreImage}
          src={props.recent.length !== 0 ? props.recent[2].imgSrc : "https://media2.giphy.com/media/3oEjI6SIIHBdRxXI40/200.gif"} />
      </div>
      <div className={style.backImage}>
        <div className={style.infoCol}>
          <span>{languageSrc.band[langProps.language]}:</span>
          <b>{props.recent.length !== 0 ? props.recent[2].band : ""}</b>
          <br />
          <span>{languageSrc.place[langProps.language]}:</span>
          <b>{props.recent.length !== 0 ? props.recent[2].place : ""}</b>
          <br />
          <span>{languageSrc.date[langProps.language]}:</span>
          <b>{props.recent.length !== 0 ? props.recent[2].date : ""}</b>
          <br />
        </div>
      </div>
    </div>
  </Col>
}
```